docs/scraping/providers/log-analytics.md

```diff
@@ -0,0 +1,41 @@
+---
+tags:
+  - Scraper
+  - Data
+---
+
+# Azure Log Analytics
+
+![Availability Badge](https://img.shields.io/badge/Available%20Starting-v2.9-green.svg)![Resource Discovery Support Badge](https://img.shields.io/badge/Support%20for%20Resource%20Discovery-No-red.svg)
+
+You can declare to scrape an Azure Log Analytics via the `LogAnalytics` resource type.
+
+The following fields need to be provided:
+
+- `name` - The name of the Azure Log Analytics
```

Comment:
Why do we need the name if we have the workspace ID?

Reply:
We need name to buildResourceUrl

Reply:
In that case, doesn't `name` and `id` feel better since they are both for the workspace?

Reply:
I just feel that `id` maybe subscriptionID or workspaceID. I prefer `workspaceName` and `workspaceId` . What do you think ?

<img width="784" alt="image" src="https://user-images.githubusercontent.com/14363009/195016384-92507380-0188-4737-8b9e-4750bcb157bd.png">